### DISCUSSION TITLE
Sitelinks search box setting not connected?

### DISCUSSION THREAD

**Discussion by timtorres** [gethyas/hyas#186] [Q&A] [answered]
There's a setting for the sitelinks search box in the params config: `siteLinksSearchBox = false` but I can't find any other reference to it in the code nor in the [SEO docs](https://gethyas.com/docs/reference-guides/seo/)... So when I set this to true, what does it do (I know what it _should_ do)? And it's not an internal Hugo variable so nothing there either.

It seemed there were a few other disconnected settings in the params file but I didn't compile them into a list.

**Comment by h-enk** [comment, marked as answer]
Removed `siteLinksSearchBox = false` (no search implementation in Hyas) + the other unused params w/ https://github.com/h-enk/hyas/pull/187
